app/src/main/java/ch/epfl/sdp/drone3d/ui/map/MissionInProgressActivity.kt

```diff
@@ -52,6 +53,11 @@ import kotlin.math.abs
 @AndroidEntryPoint
 class MissionInProgressActivity : BaseMapActivity() {
 
+    companion object {
+        private const val DEFAULT_ZOOM: Double = 17.0
```

Comment:
Each activity uses different default zooms, we could maybe try to have global parameters for the app to be more homogeneous

Reply:
We should create a class, but this can be done in another sprint